logging.json

```diff
@@ -3,7 +3,7 @@
     "disable_existing_loggers": false,
     "formatters": {
         "default": {
-            "format": "%(asctime)s [%(process)d] [%(levelname)s] [%(module)s] [%(name)s] %(message)s",
```

Comment:
asctime mag wat mij betreft ook weg... .logging gaat via systemd-journal, en die voegt ook al zijn eigen timestamps toe:
Jan 27 11:02:50 scheduler.acc.kat.prolocation.net scheduler-scheduler-app[1141203]: [2023-01-27 11:02:50 +0100] [1141203] [WARNING] [scheduler] [scheduler.schedulers.scheduler] Queue normalizer-tst is empty, not populating new tasks [queue_id=normalizer-tst, qsize=0]

Reply:
This can be handled with a custom json file if this is the case.